client/src/test/java/io/pravega/client/connection/impl/AppendBatchSizeTrackerTest.java

```diff
@@ -68,7 +68,7 @@ public void testReasonableLowerBound() {
                 tracker.recordAck(i-100);
             }
         }
-        assertTrue(tracker.getAppendBlockSize() >= size * 50);
```

Comment:
If this test is failing something is wrong.

Reply:
As the calculation has slightly changed, it looks like the append block size for this test is a bit lower than 50x, but still pretty close to it (40x or more).

Reply:
Does 45 work?

Reply:
Yes, changed.